Example/ViewController.swift

```diff
@@ -43,6 +43,7 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         oktaAppAuth = try? OktaOidc(configuration: isUITest ? testConfig : nil)
+        oktaAppAuth?.delegate = self
```

Comment:
Given that this will be open source, can we encapsulate in a view model to show best practices?

Reply:
MVVM is not required here. Example is super lightweight and fairly good shows SDK usage

Reply:
👍